### PR TITLE
Adding another user

### PR DESCRIPTION
**Pull request by juansolu**
Add example user, Juan's English twin brother.

Screen Shot 2022-09-23 at 2 37 48 PM

### CODE REVIEW

**Comment by rohithbhumar** [comment]
LGTM

**Review by Sukhpreet-Kaur01 — APPROVED** [review]

**Review by rohithbhumar — APPROVED** [review]
LGTM

**Review by vbarretoo3 — APPROVED** [review]

File: src/app/dashboard/dashboard.component.html [hunk 1]
```diff
@@ -73,6 +73,11 @@ <h4 class="card-title">Sault Git Users Stats</h4>
                                   <td><a href="https://www.linkedin.com/in/juanmoreno">juanpmoreno</a></td>
                                   <td><a href="https://github.com/juansolu">juansolu</a></td>
                               </tr>
+                              <tr>
+                                <td>John Moreno</td>
```
Comment (juansolu) [inline src/app/dashboard/dashboard.component.html:77]: Who is this person? Should he really be added?

**Review by rafaelm1234 — APPROVED** [review]

File: src/app/dashboard/dashboard.component.html (hunk 1 above)
Comment (rohithbhumar) [inline src/app/dashboard/dashboard.component.html:77]: I dont like this user

**Comment by mwiraszka** [comment]
LGTM!

**Review by vishminivarthana — APPROVED** [review]

**Comment by ghost** [comment]
I am lost

**Comment by juansolu** [comment]
This person was found to not exist. Does not need to merge.

**Pull request closed without merging**